geonode_mapstore_client/client/js/components/FiltersForm/FilterByExtent.jsx

```diff
@@ -123,22 +128,21 @@ function FilterByExtent({
                                 type: 'vector',
                                 features: [getFeatureFromExtent(queryExtent)],
                                 style: vectorLayerStyle
-                                    ? vectorLayerStyle
+                                    ? {...vectorLayerStyle, stroke: isExtentInIDL && {width: 0, color: '#397AAB'}}
                                     : {
                                         color: '#397AAB',
                                         opacity: 0.8,
                                         fillColor: '#397AAB',
                                         fillOpacity: 0.4,
-                                        weight: 4
+                                        weight: 4,
+                                        stroke: isExtentInIDL && {width: 0, color: '#397AAB'}
                                     }
                             }]
                             : []
                         )
                     ]}
                 >
-                    <ZoomTo
-                        extent={queryExtent}
-                    />
+                    {!isFiltersPanelEnabled && <ZoomTo extent={queryExtent} />}
```

Comment:
This is not having the desidered effect and it's not really needed a prop because we should update the zoom to only when we mount the component. I would suggest following changes:
- remove isFiltersPanelEnabled
- change the ZoomTo component to perform the fit action only once:
```js
function ZoomTo({
    map,
    extent
}) {
    const once = useRef();
    useEffect(() => {
        if (map && extent && !once.current) {
            const [
                aMinx, aMiny, aMaxx, aMaxy,
                bMinx, bMiny, bMaxx, bMaxy
            ] = extent.split(',');
            const projection = map.getView().getProjection().getCode();
            let bounds;
            const aBounds = reprojectBbox([aMinx, aMiny, aMaxx, aMaxy], 'EPSG:4326', projection);
            if (bMinx !== undefined && bMiny !== undefined && bMaxx !== undefined && bMaxy !== undefined) {
                const bBounds = reprojectBbox([bMinx, bMiny, bMaxx, bMaxy], 'EPSG:4326', projection);
                // if there is the second bbox we should shift the minimum x value to correctly center the view
                // the x of the [A] bounds needs to be shifted by the width of the [B] bounds
                const minx = aBounds[0] - (bBounds[2] - bBounds[0]);
                bounds = [minx, aBounds[1], aBounds[2], aBounds[3]];
            } else {
                bounds = aBounds;
            }
            map.getView().fit(bounds, {
                size: map.getSize(),
                duration: 300
            });
            // ensure to avoid other fit action by setting once to true
            once.current = true;
        }
    }, [ extent ]);

    return null;
}
```

The expectation is to see the bbox centered onec we open again the filter panel.
Note that the computation of bounds has changed in the snippet so please verify that everything is working